docker/app/docker-compose.yaml

```diff
@@ -9,8 +9,8 @@ x-app: &app
     dockerfile: docker/app/Dockerfile
     args:
       BUILD_TAG: 3.1.6-alpine3.20
-      USER_ID: ${USER_ID:-1001}
-      GROUP_ID: ${GROUP_ID:-1001}
+      USER_ID: ${USER_ID:-1000}
+      GROUP_ID: ${GROUP_ID:-1000}
```

Comment:
Is this going to break some development environments?

Reply:
I'll just change it back.